locale/en/LC_MESSAGES/django.po

```diff
@@ -1848,8 +1848,8 @@ msgstr ""
 
 #: authentik/providers/oauth2/models.py
 msgid ""
-"Tokens not valid on or after current time + this value (Format: "
-"hours=1;minutes=2;seconds=3)."
+"Tokens not valid on or after current time + this value (Format: hours=1;"
+"minutes=2;seconds=3)."
```

Comment:
@rissson I swear I've seen a few PRs that always contain this line with nothing else 🤔

https://github.com/goauthentik/authentik/pull/18046/files#diff-c9557c19413e04ae10a25832b30f73ee68d4e9bdae31b49f5404b9086b80f126L1744

Does this file get formatted somewhere?